ui/src/app/pages/data/data-set/dataset-page.tsx

```diff
@@ -1372,6 +1356,16 @@ export const DatasetPage = fp.flow(
                 PrePackagedConceptSetEnum.FITBITHEARTRATELEVEL
               );
               break;
+            case PrepackagedConceptSet.FITBITSLEEPDAILYSUMMARY:
+              selectedPrePackagedConceptSDetEnum.push(
+                PrePackagedConceptSetEnum.FITBITSLEEPDAILYSUMMARY
+              );
+              break;
+            case PrepackagedConceptSet.FITBITSLEEPLEVEL:
+              selectedPrePackagedConceptSDetEnum.push(
+                PrePackagedConceptSetEnum.FITBITSLEEPLEVEL
+              );
+              break;
```

Comment:
Can we also apply this pattern to this switch statement as well?

Reply:
Couldn't get the same conversion to work for this enum, kept complaining about the types. After trying for a while, I realized both conversions don't seem necessary, so I changed all handling of prepackaged sets to use `PrePackagedConceptSetEnum` and deleted `PrepackagedConceptSet` and all conversion functions. Seems to be working properly, if you can give it another run through to verify.